tests/test-01-failure.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if gitolize.sh -r "file://$GIT_REPOSITORY" false
+then
+  echo "expected failure" 1>&2
```

Comment:
Shall we also test a case where a gitolized command fails but the changes are still commited and pushed to the repository?

Reply:
You knew it, didn't you ;)!

Fixed and tested in 6093b436f116d5fac562e548eb57c5bf91daa332